contracts/NewfiAdvisor.sol

```diff
@@ -173,25 +269,33 @@ contract NewfiAdvisor is Initializable, ReentrancyGuardUpgradeSafe, ProxyFactory
         address _advisor,
         uint256 _stableProportion,
         uint256 _volatileProportion
-    ) external payable {
+    ) external payable nonReentrant {
```

Comment:
Did you get the tests working with Reentrancy? I know that its nice to have but since we're just running this on a demo on our local machines it doesn't seem like we have to worry about an reetrant attack at the moment